src/components/AdminPanel/Sections/AddHrForm.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { useForm, SubmitHandler } from "react-hook-form";
+import {toast} from "react-toastify";
+
+type FormInputs = {
+  email: string;
+  fullName: string;
+  company: string;
+  maxReservedStudents: number;
+};
+export function AddHrForm() {
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<FormInputs>();
+
+  const onSubmit: SubmitHandler<FormInputs> = async ({email, fullName, company, maxReservedStudents}) => {
+    try {
+      const res = await fetch('/hr', {
+        method: 'POST',
+        mode: 'cors',
+        headers: {
+          "Access-Control-Allow-Origin":"true",
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          email,
+          fullName,
+          company,
+          maxReservedStudents: Number(maxReservedStudents),
+        }),
+      });
+      const data = await res.json();
+      if (data.ok) {
+        toast.success(data.message);
+      } else {
+        toast.error(data.message);
+      }
+    } catch (e) {
+      console.log(e);
+    }
+  }
+
+  return (
+    <div className="addHrFormContainer">
+      <form className="addHrFormContainer__form" onSubmit={handleSubmit(onSubmit)}>
+        <h2>Dodaj HR</h2>
+        <input
+          {...register("email", { required: true, maxLength: 255 })}
+          type="email"
+          placeholder="Email"
+        />
+        {errors.email?.type === "required" && "To pole jest wymagane"}
```

Comment:
Zastanowiłbym się nad jakimś chociaż wydzieleniem takich informacji do osobnej funkcji/komponentu. Bo to trochę zaciemnia obraz.